components/atoms/ActionButton.js

```diff
@@ -17,12 +18,13 @@ export function ActionButton({
 }) {
   //Styling for buttons and links
   const defaultStyle =
-    'rounded-sm focus:ring-1 focus:ring-black focus:ring-offset-2 py-2 px-4 bg-custom-blue-blue text-white border border-custom-blue-blue active:bg-custom-blue-dark hover:bg-custom-blue-light'
+    'font-display rounded-sm focus:ring-1 focus:ring-black focus:ring-offset-2 py-2 px-4 bg-deep-blue-solid text-white border border-deep-blue-solid active:bg-deep-blue-dark hover:bg-bright-blue-dark'
 
   return href ? (
     <Link href={href}>
       <a
-        className={`${!className ? defaultStyle : className}`}
+        className={`${defaultStyle} 
+        ${extendedClass && !className ? extendedClass : ''}`}
```

Comment:
This line does not do what you expect. className is never executed

Reply:
Please help me fix it.